daemons/compute/server/servers/server_default.go

```diff
@@ -317,8 +317,10 @@ func (s *defaultServer) List(ctx context.Context, req *pb.DataMovementListReques
 	}
 
 	list := dmv1alpha1.RsyncNodeDataMovementList{}
-	if err := s.client.List(ctx, &list, opts...); err == nil {
-		rsyncdm_uids := []string{}
+	if err := s.client.List(ctx, &list, opts...); err != nil {
+		return nil, err
+	} else {
```

Comment:
No need for this to be in an `else{}` block since the above is a return. This might seem like nit-picking, but it makes things easier to read - you just find your column and follow it down vertically,  knowing everything indented further is exception cases.

Reply:
Fixed.